### PR TITLE
fix: colorspace build fix

### PR DESCRIPTION
**Pull request by demisquare**
This PR fixes the ColorSpace missing field error during build of the library, by adding a <ImageData> assert type to variables, and updates TensorFlow to 3.18.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@mediapipe/face_mesh": "^0.4.1633559619",
     "@msgpack/msgpack": "^2.7.2",
     "@techstark/opencv-js": "^4.5.5-release.2",
-    "@tensorflow/tfjs": "^3.17.0",
+    "@tensorflow/tfjs": "^3.18.0",
     "@tensorflow/tfjs-backend-webgl": "^3.17.0",
```
Comment (krsbx) [inline package.json:57]: please change the backend webgl to 3.18 as well

Comment (krsbx) [inline package.json:57]: please change the backend webgl to 3.18 as well

**Comment by demisquare** [comment]
Did you need any other changes?

**Review by krsbx — APPROVED** [review]
LGTM

**Pull request merged**